src/main/java/at/hannibal2/skyhanni/config/Features.java

```diff
@@ -32,9 +33,9 @@
 
 
 public class Features extends Config {
-    public static final ResourceLocation DISCORD = new ResourceLocation("notenoughupdates:social/discord.png");
-    public static final ResourceLocation GITHUB = new ResourceLocation("notenoughupdates:social/github.png");
-    public static final ResourceLocation PATREON = new ResourceLocation("notenoughupdates:social/patreon.png");
+    public static final MyResourceLocation DISCORD = new MyResourceLocation("notenoughupdates", "social/discord.png");
+    public static final MyResourceLocation GITHUB = new MyResourceLocation("notenoughupdates", "social/github.png");
+    public static final MyResourceLocation PATREON = new MyResourceLocation("notenoughupdates", "social/patreon.png");
```

Comment:
while we are changing these lines can you please make these no longer use the neu resources and instead copy these resource files into skyhanni and use those

Reply:
done <3 thanks cal